### ISSUE TITLE
FAQ lists obsoloete / no longer produced cables

### ISSUE THREAD

**Issue by laf0rge** [blackmagic-debug/blackmagic#1166]
This is just FYI:

The two parts listed in the FAQ (both the old github wiki and the new black-magic.org) list pre-crimed cables that are obsoloete and no longre produced:

> The UART cable can also be built with pre crimped wires:
>   [Digi-key](https://www.digikey.com/product-detail/en/molex-llc/06-66-0012/WM2319-ND/2405453) red 150mm pre crimped cable
>   [Digi-key](https://www.digikey.com/product-detail/en/molex-llc/06-66-0013/WM2320-ND/2405456) red 300mm pre crimped cable

**Comment by esden** [comment]
You are correct. There is a much bigger selection now. https://www.digikey.com/short/08n0q938 I will update the documentation. I am moving this issue to the website repo where it belongs.
